src/main/java/mcjty/theoneprobe/apiimpl/providers/DefaultProbeInfoProvider.java

```diff
@@ -188,10 +190,15 @@ private void showRF(IProbeInfo probeInfo, World world, BlockPos pos) {
             if (handler != null) {
                 addRFInfo(probeInfo, config, handler.getEnergyStored(), handler.getMaxEnergyStored());
             }
+        } else if (Loader.isModLoaded("tesla") && te != null && te.hasCapability(TeslaCapabilities.CAPABILITY_HOLDER, null)) {
```

Comment:
Used a WIP mod of mine, disabled the ForgeEnergy support, leaving only Tesla.
http://imgur.com/a/akKOz

Confirmed that the first case does not crash, no errors in console whatsoever. Second case it correctly find the build-in energy capabity with Tesla (confirmed with logging).